vrtUtil/vrtUtil.js

```diff
@@ -0,0 +1,110 @@
+const path = require('path');
+const fs = require('fs');
+const ImageDiff = require('./ImageDiff');
+
+// Matching threshold as a decimal percentage of the maximum acceptable square distance between two colors;
+// Ranges from 0 to 1. Smaller values make the comparison more sensitive.
+const DEFAULT_VRT_THRESHOLD = 0.1;
```

Comment:
This value is a decimal fraction but the comparison in `ImageDiff` is denoted as a percentage

Reply:
so this value is actually the threshold that is passed into the `pixelmatch` comparison algorithm, and isnt a threshold directly for the passing percentage, which could be confusing.  In `pixelmatch` the threshold is not exactly a minimum percentage of difference, its the percentage of the maximum acceptable square distance between two colors which i commented above. 
 ref: https://github.com/mapbox/pixelmatch/blob/ced69a33f0a3c8e9377a221e274693690f69be22/index.js#L45
 
 I think what I can do is rename this var to be more accurate, as well as create another var for a maximum difference percentage threshold